internal/cloudconfig/instancecfg/instancecfg.go

```diff
@@ -351,8 +355,9 @@ type StateInitializationParams struct {
 
 type stateInitializationParamsInternal struct {
 	AgentVersion                            string                            `yaml:"agent-version"`
-	ControllerConfig                        map[string]interface{}            `yaml:"controller-config"`
-	ControllerModelConfig                   map[string]interface{}            `yaml:"controller-model-config"`
+	ControllerConfig                        map[string]any                    `yaml:"controller-config"`
+	ControllerModelConfig                   map[string]any                    `yaml:"controller-model-config"`
+	ControllerModelAuthorizedKeys           []string                          `yaml:"controller-model-authorized-keys"`
```

Comment:
This change will likely be in the next PR? I think it might be better just to remove this change and add it to the next delta.